apps/sms/test/marionette/lib/views/new-message/view.js

```diff
@@ -167,6 +167,12 @@ NewMessageView.prototype = {
     return this.composerAccessors.messageInput.scriptWith(function(el) {
       return document.activeElement === el;
     });
+  },
+
+  isRecipientsInputFocused: function() {
+    return this.accessors.recipientsInput.scriptWith(function(el) {
+      return document.activeElement === el;
+    });
```

Comment:
Yeah, would be great to have shared tools for that and we don't have them now :/ What about ```view_tools (or better name).js``` in [1]?

[1] https://github.com/mozilla-b2g/gaia/tree/master/apps/sms/test/marionette/lib/views/shared